functions/Remove-DbaDbMailAccount.ps1

```diff
@@ -0,0 +1,122 @@
+        dbatools PowerShell module (https://dbatools.io)
+        Copyright: (c) 2021 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaDbMailAccount
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbMailAccount -SqlInstance localhost, localhost\namedinstance
+
+        Removes all database mail accounts on the localhost, localhost\namedinstance instances.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbMailAccount -SqlInstance localhost -Name MyDatabaseMailAccount
+
+        Removes MyDatabaseMailAccount database mail account on the localhost.
+
+    .EXAMPLE
+        PS C:\> Get-DbaDbMailAccount -SqlInstance SRV1 | Out-GridView -Title 'Select database mail account(s) to drop' -OutputMode Multiple | Remove-DbaDbMailAccount
+
+        Using a pipeline this command gets all database mail accounts on SRV1, lets the user select those to remove and then removes the selected database mail accounts.
+
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    param (
+        [Parameter(ParameterSetName = 'NonPipeline', Mandatory = $true, Position = 0)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [Parameter(ParameterSetName = 'NonPipeline')]
+        [PSCredential]$SqlCredential,
+        [Parameter(ParameterSetName = 'NonPipeline')]
+        [string[]]$Name,
+        [Parameter(ParameterSetName = 'NonPipeline')]
+        [string[]]$ExcludeAccount,
+        [parameter(ValueFromPipeline, ParameterSetName = 'Pipeline', Mandatory = $true)]
+        [Microsoft.SqlServer.Management.Smo.Mail.MailAccount[]]$InputObject,
+        [Parameter(ParameterSetName = 'NonPipeline')][Parameter(ParameterSetName = 'Pipeline')]
+        [switch]$EnableException
+    )
+
+    begin {
+        $dbMailAccounts = @( )
+    }
+
+    process {
+        if ($SqlInstance) {
+            $params = $PSBoundParameters
+            $null = $params.Remove('WhatIf')
+            $null = $params.Remove('Confirm')
+            $dbMailAccounts = Get-DbaDbMailAccount @params
+        } else {
+            $dbMailAccounts += $InputObject
+        }
+    }
+
+    end {
+        # We have to delete in the end block to prevent "Collection was modified; enumeration operation may not execute." if directly piped from Get-DbaDbMailAccount.
+        foreach ($dbMailAccount in $dbMailAccounts) {
+            if ($PSCmdlet.ShouldProcess($dbMailAccount.Parent.Parent.Name, "Removing the database mail account $($dbMailAccount.Name) in the database $($dbMailAccount.Parent.Name) on $($dbMailAccount.Parent.Parent.Name)")) {
+                $output = [pscustomobject]@{
+                    ComputerName  = $dbMailAccount.Parent.Parent.ComputerName
+                    InstanceName  = $dbMailAccount.Parent.Parent.ServiceName
+                    SqlInstance   = $dbMailAccount.Parent.Parent.DomainInstanceName
+                    DbMailAccount = $dbMailAccount.Name
```

Comment:
```suggestion
                    Name = $dbMailAccount.Name
```
Matching the output we have for Get and New commands.

Reply:
Done. Changed to match outputs of the Get / New commands.